src/handlers/project/create.ts

```diff
@@ -59,21 +60,38 @@ async function createProjectRole(
   return role;
 }
 
+async function sortCategoryChannels(
+  guild: Guild,
+  categoryId: Snowflake
+): Promise<void> {
+  const category = await guild.channels.fetch(categoryId);
+  if (category instanceof CategoryChannel) {
+    const categoryChannels = [...category.children.values()];
+    categoryChannels.sort((a, b) => (a.name < b.name ? -1 : 1));
+    for await (const [index, channel] of categoryChannels.entries()) {
+      await channel.setPosition(index);
+    }
+  }
+}
```

Comment:
This could be a good one to pull out for reuse, it seems like this would be beneficial as a sort of standalone slash command as well for mods for sorting when needed? Just a thought.

Reply:
That's a great idea! Although at this point I'm not sure if the mods frequently sort anything besides the IC/GB channels alphabetically.

Also I just realized that `for await` loop is terribly inefficient since it's awaiting each promise synchronously in series. Lemme just patch that up and I'll merge this.

Reply:
Weirdly something went wrong when I tried to fire off the requests in parallel, I would suspect that the Discord.js library is not throttling the requests well enough to meet the rate limits of the Discord API. For now I'll keep it as requests in sequence.

Reply:
Good catch on the reminder comment, somehow overlooked that. Yeah performance wise I wasn't too concerned there given the use case but yeah can look to address that at a later point

Reply:
I can't recall a time in the last year where there was a crazy amount of concurrent channels in either, not to say that can't happen